tecken/gunicornhooks.py

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""
+Gunicorn server hooks.
+
+See https://docs.gunicorn.org/en/stable/settings.html#server-hooks
+"""
+
+import markus
+
+from tecken import settings
```

Comment:
This doesn't pull in any actual Django machinery, right? I'm thinking about whether this might be to heavy-weight to pull into gunicorn, but I don't think it is.

Reply:
That's correct. This doesn't pull in any Django machinery. I'll a note about that at the top of the file.